src/components/Satelites/Moons.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class MoonsList extends React.Component {
```

Comment:
MoonList, not MoonsList ;)

and file should be named after the component it declares